src/CollAction/Views/Projects/Find.cshtml

```diff
@@ -9,118 +9,7 @@
     var defaultDisplayModel = FindProjectViewModel.DefaultDisplayProjectViewModel;
 }
 
-<h2>@Localizer["Find"]</h2>
```

Comment:
I'd prefer the functionality stays in working condition in the master branch, even if it's due to be replaced with react.